fdbclient/BackupContainerFileSystem.actor.cpp

```diff
@@ -637,6 +724,49 @@ class BackupContainerFileSystemImpl {
 				    (desc.contiguousLogEnd.get() - 1) > desc.maxRestorableVersion.get())
 					desc.maxRestorableVersion = desc.contiguousLogEnd.get() - 1;
 			}
+
+			// If there is logs gap after contiguousLogEnd and if current snapshot is restorable(have continuous logs)
+			if (desc.contiguousLogEnd.present() && desc.contiguousLogEnd.get() <= s.beginVersion &&
+			    s.restorable.get()) {
+				if (desc.minRestorableVersion.present() && desc.maxRestorableVersion.present()) {
+					ASSERT(desc.minRestorableVersion.get() < s.beginVersion);
+
+					// check if we have contiguous logs from minRestorableVersion to current snapshot endVersion
+					bool contiguousLogs = false;
+					if (desc.partitioned)
+						contiguousLogs =
+						    isPartitionedLogsContinuous(logs, desc.minRestorableVersion.get(), s.endVersion);
+					else
+						contiguousLogs =
+						    hasContinuousLogsForSnapshot(logs, desc.minRestorableVersion.get(), s.endVersion);
+
+					if (contiguousLogs) {
+						// The previous restorable version can be extended to current snapshot vesion,
```

Comment:
nits: s/vesion/version

Reply:
I will take this a TO-Do item in my next PR. Just, don't want to delay this PR anymore.